enhancements/network/ovn-kubernetes-evpn.md

```diff
@@ -0,0 +1,429 @@
+---
+title: ovn-kubernetes-evpn
+authors:
+  - "@jcaamano"
+reviewers:
+  - "@anuragthehatter"
+  - "@jechen0648"
+  - "@kyrtapz"
+  - "@maiqueb"
+  - "@mattedallo"
+  - "@pperiyasamy"
+  - "@tssurya"
+approvers:
+  - "@tssurya"
+api-approvers:
+  - None
+creation-date: 2025-10-14
+last-updated: 2025-10-24
+tracking-link:
+  - https://issues.redhat.com/browse/CORENET-6429
+see-also:
+  - https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5089
+
+---
+
+# OVN-Kubernetes support for EVPN
+
+## Summary
+
+This feature allows exposing primary Cluster User Defined Networks (P-CUDNs)
+externally via a VPN to other entities either inside, or outside the cluster;
+using BGP and EVPN as the common and native networking standard that will enable
+integration into user networks without SDN specific network protocol
+integration, and providing an industry standardized way to achieve network
+segmentation between sites.
+
+This enhancement is aligned and being worked on in tandem to a corresponding
+OVN-Kubernetes upstream [enhancement][1]. As such, there will be references to
+it for much of the content. The intention of this enhancement is to outline the
+necessary changes to consume and integrate that functionality in OCP, including
+the interaction with the Cluster Network Operator (CNO) and our test plan for
+this feature. However, in case the upstream enhancement is found to be
+inadequate, one of the following outcomes is possible depending on the
+circumstances of the inadequacy and the best context to resolve it:
+* Amend the upstream enhancement if still open.
+* Work on a follow up upstream enhancement while keeping this one open.
+* Work on a new downstream enhancement that either replaces or follows this one. 
+
+[1]: https://github.com/ovn-kubernetes/ovn-kubernetes/pull/5089
+
+
+## Motivation
+
+The motivation for this feature is aligned to the one described in the upstream
+[enhancement][2].
+
+[2]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#introduction
+
+### User Stories
+
+The user stories for this feature are aligned to the ones described in the
+upstream [enhancement][3].
+
+[3]: https://github.com/ovn-kubernetes/ovn-kubernetes/blob/4568650412f85cdcf7671c74d2e9b3bb67d3b366/docs/okeps/okep-5088-evpn.md#user-storiesuse-cases
+
+### Goals
+
+The goals for this feature are aligned to the ones described in the upstream
```

Comment:
I just noticed that in the upstream enhancement its mentioned:

```
Support for interconnecting two Kubernetes clusters with EVPN and then allowing VM migration across them.
```
as a future goal.

and while that's OK for upstream for us the immediate goal in downstream is eventually for all our pieces to come together to make that happen.
Are we not aiming to do that in this enhancement - meaning is that staying as a future goal? maybe good to clarify if the future goals also stay the same as is in upstream enhancement?